app/controllers/api/v1/steps_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class StepsController < BaseController
+      before_action :load_team, :load_project, :load_experiment, :load_task, :load_protocol
```

Comment:
I find it a bit cumbersome that user has to provide all of the above ids where task or protocol id would suffice. But I guess this is how it is.

Reply:
Yeah, it's quite annoying, but I just follow the nested schema for resources we have. 🤔

Reply:
Any ideas @jbargu ?

Reply:
Probably best to not reinvent the wheel and just be consistent. This should be a part of a bigger debate based on user feedback.